pkg/zcmd/zpool/replace/builder.go

```diff
@@ -166,8 +166,8 @@ func (p *PoolDiskReplace) Build() (*PoolDiskReplace, error) {
 	}
 
 	p.appendCommand(&c, p.Pool)
-	p.appendCommand(&c, fmt.Sprintf(" %s", p.OldVdev))
-	p.appendCommand(&c, fmt.Sprintf(" %s", p.NewVdev))
+	p.appendCommand(&c, " "+p.OldVdev+" ")
+	p.appendCommand(&c, " "+p.NewVdev+" ")
```

Comment:
Since old command doesn't include space after oldVded & NewVdev let's also maintain the same.
```suggestion
	p.appendCommand(&c, " "+p.OldVdev)
	p.appendCommand(&c, " "+p.NewVdev)
```